docs/data-model.asciidoc

```diff
@@ -117,7 +117,7 @@ A transaction contains:
 * Data about the environment in which the event is recorded:
 ** Service - environment, framework, language, etc.
 ** Host - architecture, hostname, IP, etc.
-** Process - args, PID, PPID, etc.
+** Process - arguments, PID, PPID, etc.
```

Comment:
Can we ignore this one? The field is `process.args` so I think it makes more sense to keep args here.